gramps/gen/utils/grampstranslation.py

```diff
@@ -292,7 +293,7 @@ class GrampsNullTranslations(gettext.NullTranslations):
     neither will be the returned string.
     """
 
-    def gettext(self, msgid, context=""):
+    def gettext(self, msgid, context="") -> str:
```

Comment:
```suggestion
    def gettext(self, msgid: str, context: str = "") -> str:
```
Missed typing the parameters?

Reply:
Thanks. I'd originally tried to types to methods called by gettext but had to step back. I went back a bit too far.